jetty-home/src/main/resources/modules/logging-slf4j.mod

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Configure logging to use slf4j with no impl
+(If you don't select an impl, then NOP will be used by slf4j)
+
+[tags]
+logging
+
+[provides]
+slf4j
+
+[lib]
+lib/logging/slf4j-api-${slf4j.version}.jar
+
+[ini]
+slf4j.version?=2.0.0-alpha1
+jetty.webapp.addSystemClasses+=,org.slf4j.
```

Comment:
I think that slf4j should be a server class not a system class.   There is no reason that a webapp cannot have it's own version of slf4j which is a different version.     Now we might want to provide a mechanism that exposes server slf4j to a webapp, but it should not be the default.

Reply:
This needs a review.

Reply:
Behavior here could be a new module - default behavior - option to expose to webapp.

Reply:
Changed to server classes in this PR.